tests/testthat/test-multitables.R

```diff
@@ -363,16 +374,59 @@ with_mock_crunch({
         expect_header(
             expect_POST(
                 tabBook(m, data = ds2[c("gender", "starttime")]),
-                "https://app.crunch.io/api/datasets/3/multitables/ed30c4/tabbook/"
+                "https://app.crunch.io/api/datasets/3/multitables/ed30c4/export/",
+                '{\"filter\":null,',
+                '\"weight\":"https://app.crunch.io/api/datasets/3/variables/birthyr/",',
+                '\"options\":[],"where":{"function":"select","args":[{"map":',
+                '{"66ae9881e3524f7db84970d556c34552":',
+                '{"variable":"https://app.crunch.io/api/datasets/3/variables/gender/"},',
+                '"d7c21314ca9e453c93069168681a285c"',
+                ':{"variable":"https://app.crunch.io/api/datasets/3/variables/starttime/"}}}]}'
             ),
             "Accept: application/json"
         )
     })
 
-    ## TODO: test the query shape
+    test_that("tabBook with options", {
+        expect_POST(
+            tabBook(mults[[1]],
+                    data = ds, output_format = "json", format = list(pval_colors = TRUE))
+            ,
+            "https://app.crunch.io/api/datasets/1/multitables/ed30c4/export/",
+            '{\"filter\":null,\"weight\":null,\"options\":{"format":{"pval_colors":true}}}'
+        )
+    })
+
+    test_that("tabBook warning when using format argument", {
```

Comment:
@malecki test for deprecation warning for `format="json"`. Currently only on new endpoint, mostly because I wasn't sure how testthat handles two warnings (and there will be one for old endpoint and another for old argument). Do you think it's worth adding that case?

Reply:
nah. Let’s merge it and then fix https://github.com/Crunch-io/crunchtabs/blob/master/R/tabBooks.R#L11 to use `output_format`
cc @1beb — this should be mostly transparent to you; expect deprecation warnings in crunchtabs and a PR to fix them. Maybe we should consider pinning an rcrunch version in https://github.com/Crunch-io/crunchtabs/blob/master/DESCRIPTION#L19?

Reply:
I am indeed getting a tonne of the deprecation warnings but everything appears to be working fine for PDF outputs. I've tried about 5 different clients stuff so far and no failures. I still need to test ExcelTabs because that is used heavily by YouGov Blue. I have examples collated so the remaining step is a bit of comparative code and we should be good to go.

Reply:
To confirm is it only the `format` argument that needs to be changed to `output_format`. We are working with two different branches so I will need to update both. Has the option stayed the same "json"?

Reply:
Yep, `"json"` and `"xlsx"` are still the two options, but we had to rename the argument to avoid a conflict with options in the new endpoint.

That's the only thing you should have to change.